CorsixTH/Lua/world.lua

```diff
@@ -2604,6 +2605,15 @@ function World:afterLoad(old, new)
 
   self.earthquake:afterLoad(old, new)
 
+  -- Fix VIPs to not show mood on hover
+  for _, entity in ipairs(self.entities) do
+    if class.is(entity, Humanoid) and entity:isType() then
+      if entity:isType("VIP") or entity:isType("Inspector") then
+        entity.hover_moods = false
+      end
+    end
+  end
+
```

Comment:
This may be cleaner with my comment above (note, for the afterload you need to wrap in an `old < xyz` where xyz is the new savegame version number you set in `app.lua` (the variable called `local SAVEGAME_VERSION = xxx -- Some comment you update`. For new version, just bump the current number in master by 1.

```lua
if old < xyz then
  for _, entity in ipairs(self.entities) do
    if class.is(entity, Humanoid) then
      entity:setIfHoverMoodsVisible()
    end
  end
end
```

^ For the above, `ipairs` loop is well used on `self.entities`. `pairs` is safer but shouldn't be necessary.